app/views/shared/_default_client_settings.html.erb

```diff
@@ -15,9 +15,10 @@
   settings[:canvas_callback_url]  = user_canvas_omniauth_callback_url(host: Integrations::CanvasApiSupport.oauth_host)
   settings[:oauth_consumer_key] = current_application_instance.lti_key
   settings[:canvas_auth_required] = @canvas_auth_required
-  if @canvas_auth_required
-    settings[:authorization] = default_jwt
-  end
+
+  # always include this token in case the client needs to re-auth
+  settings[:authorization] = default_jwt
```

Comment:
Would there be a reason to give someone not logged in a jwt? Or should this be moved into the `signed_in?` check?

Reply:
You're right.  We shouldn't need to send this at all since it's the same as `DEFAULT_JWT` but the atomic_canvas gem currently needs it in the settings.  I think we should update `CanvasAuthentication` to accept an authorization parameter and get rid of this.  For now I only add it if signed in

Reply:
We've decided to fix this by updating atomic-canvas:
https://github.com/atomicjolt/atomic-canvas/pull/39

If the jwt exists it will be automatically added to the form.